src/bp-core/compatibility/bp-incompatible-plugins-helper.php

```diff
@@ -154,6 +154,15 @@ function bp_core_instructor_role_post_exclude( $exclude_posts ) {
 
 		add_filter( 'wdmir_exclude_post_types', 'bp_core_instructor_role_post_exclude', 10, 1 );
 	}
+
+	/**
+	 * Include filters when Woocommerce plugin is activated
+	 *
+	 * Support Woocommerce
+	 */
+	if ( class_exists( 'WooCommerce' ) ) {
+		require buddypress()->compatibility_dir . '/bp-woocommerce-helpers.php';
```

Comment:
```suggestion
		require buddypress()->compatibility_dir . '/class-bb-woocommerce-helpers.php';
```
Also rename the class file as well.